src/rtp.c

```diff
@@ -201,13 +201,6 @@ rtp_check_packet(packet_t *packet)
         if (!(stream_is_complete(stream))) {
             stream_complete(stream, src);
             stream_set_format(stream, format);
-            // Check if an stream in the opposite direction exists
-            if (!(reverse = rtp_find_call_stream(stream->media->msg->call, stream->dst, stream->src))) {
-                reverse = stream_create(stream->media, stream->src, PACKET_RTP);
-                stream_complete(reverse, stream->dst);
-                stream_set_format(reverse, format);
-                call_add_stream(msg_get_call(stream->media->msg), reverse);
-            }
```

Comment:
Aha, so this covers the so called 'comedia' mode which is used by Asterisk for example.., I get it.

The inconsistence I see is that the same `media` is set to both streams (original and reverse). And that, I think, makes or can make the RTP arrows and/or ports to be wrongly displayed. In the SDP media parser the `media` associated to a stream is that containing its description.